src/index.html

```diff
@@ -6,7 +6,314 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <div class="logo">
+        <a href="#home">
```

Comment:
Do you really need a div container for logo? I think you can set all the properties to link directly